addon/instance-initializers/hot-loader-livereload-plugin.js

```diff
@@ -15,22 +36,10 @@ function createPlugin (appName, hotReloadService) {
       // Reloading app.js will fire Application.create unless we set this.
       // TODO: make sure this doesn't break tests
       window.runningTests = true;
-      var tags = document.getElementsByTagName('script');
-      for (var i = tags.length; i >= 0; i--){
-        if (tags[i] && tags[i].getAttribute('src') != null && tags[i].getAttribute('src').indexOf(appName) !== -1) {
-          tags[i].parentNode.removeChild(tags[i]);
-        }
-      }
-      var script = document.createElement('script');
-      script.onload = function() {
-        setTimeout(function() {
-          window.runningTests = false;
-          hotReloadService.trigger('willHotReload', path);
-        }, 10);
-      };
-      script.type = 'text/javascript';
-      script.src = `/assets/${appName}.js`;
-      document.body.appendChild(script);
+      all([reloadScript(`/assets/${appName}.js`), reloadScript('/assets/vendor.js')]).then(()=>{
```

Comment:
Unfortunately, it still didnt work. We cant load vendor.js twice witout blowing away loader.js's eak_seen. Ill investigate later this week and check with engines lazy loading because we are doing something similar.